src/configure.py

```diff
@@ -150,9 +150,7 @@ def onrewardstype(input):
         rt = RewardsType(input.lower())
         parser.set(REWARDS_TYPE, str(rt))
```

Comment:
@nicolasochem don't we have to drop the  `RewardsType:ESTIMATED` from `Constants.py` ???

Reply:
You tell people to only use the two supported types but if someone types "estimated" you allow it and catch the error later in the parser, right?

Reply:
aaaand you need the type because you set it later to ESTIMATED when cecking for [-11,-6] RELEASE_OVERRIDE. ok ok ok

Reply:
Yes, correct. `ESTIMATED` is still used internally (and visible in calculations.csv) but I made it impossible to configure it in the config file.